### DISCUSSION TITLE
Can I control a VRV system?

### DISCUSSION THREAD

**Discussion by villamanuel1980** [revk/xESP32-Faikout#122] [Q&A] [answered]
Hello! We want to use Home Assistant to control the A/C in a small hotel. It's 21 rooms with a Daikin VRV system. Each room has a BRC2C51 controller now. Then there is a central unit that reports status on each room and lets you set limits and so on.

Can we do this with Faikin or Faikin 2?

Thanks! Sorry if this is too basic a question, we're new to VRV systems and still trying to learn.

**Comment by revk** [comment, marked as answer]
Basically, no way to know for sure.

If the units have an S21 connector then probably.

And if they do, I'd suggest trying one Faikin and see if it works, first.

If it does Home Assistant should make it easy.

**Reply by villamanuel1980** [reply]
Thanks! So we will buy one and look for that connector in the unit for each room, right? no way to operate over the centralized unit?

Buying 21 Faikin is no problem, just trying to understand the "right" way to do this. We just basically want the A/C to turn of when the room is empty, automatically.

Should we wait for Faikin 2 to be ready or will it make no difference?

Thanks for your help! Ordering right away!

**Reply by revk** [reply]
I would assume each unit has an S21. Check first before buying. If it does the Faikin would allow them to all be controlled via Home Assistant.

Faikin2 is not a thing. I have done several versions of PCB, and there was Faikin2 and Faikin3. But the latest design is what is being sold now on Amazon. The only variation is there are some ESP32-S3, but they have the same Tx/Rx connection circuit, so work the same way.

**Reply by villamanuel1980** [reply]
Understood!

I will check the connections inside the unit and proceed accordingly.
Will this let us keep both the remote control and the clients control unit functional? I struggle to understand what will happen if the client leaves it on but the mqtt command is to turn off?

Ordering from Amazon now to test.
We will keep you posted on progress. If it works, it will be a great leap forward for us! VRVs have always been a pain.

Thanks for your patience and speed! :)

**Reply by revk** [reply]
Whatever command is sent last applies. Remote still works, but you can decide to then send an MQTT to override.

Remotes have an annoying memory so if they thing on, then power button sends off, even if already off, which may confuse people a tad.

**Reply by villamanuel1980** [reply]
Hello!

So I received my Faikin but I'm still doubtful it could work. These are the units we have:

- Enfriadora Ciatesa IWEB 600 (S/N. 090201110E).
- VRV Daikin RXYQ16P7W1BA (S/N. 1700334).
- VRV Daikin RXYQ16P7W1BA (S/N. 1700390).
- VRV Daikin RXYQ10P7W1B (S/N. 1707121).
- Cassette Daikin FXZQ20M8V1B (S/N. A16403184464) / 1 unidad en despacho Dirección.
- Cassette Daikin BYFQ60B8W1 (S/N. 2721379) / 1 unidad en recepción.
- Conductos Daikin FXSQ63M8V3B / 6 unidades en zonas comunes.
- Conductos Daikin FXSQ32M8V3B / 21 unidades en habitaciones.
- Conductos Daikin FXSQ63M8V3B / 2 unidades en restaurante.
- Conductos Daikin FXSQ32M8V3B / 2 unidades en bar.

If I understand correctly, I could operate the 31 units FXSQ32M8V3B  with Faikin and they'd report their operation back to the VRV central unit so it can behave accordingly? Did I get this right?

Thanks!!

**Reply by revk** [reply]
It is no use asking me about specific models - anyone else here with those models that can comment?

Anything that has S21 should work, and then could be controller by, for example, Home Assistant, centrally. Or your own MQTT tools.

Best to get one Faikin and see what works.

**Comment by villamanuel1980** [comment]
Thanks! Super helpful. Can't wait to try it! :)

**Comment by villamanuel1980** [comment]
Thanks! Sorry I thought you might now.

I'll report back in case it can be of help to anyone else!

Cheers!

**Reply by revk** [reply]
Basically I have only two models here that I use, and I have been working with customers as needed to adjust things to handle a variety of other models. There is a wiki with a list of models people have tried (feel free to update it).

**Reply by villamanuel1980** [reply]
Is there any way we can log what the Faikin reads from the pins so it can be of help to you? Either to identify if it's indeed an S21 port or not, or if VRVs have anything else that is different?

**Comment by dzungpv** [comment]
@villamanuel1980 do your VRVs working with Fakin?

**Reply by villamanuel1980** [reply]
Not sure yet. There's a promising port but couldn't make it work for the moment.

**Comment by villamanuel1980** [comment]
Still untested, funnyliy enough, we've set a date with the client to test
tomorrow! I'll keep you posted!

**Reply by dzungpv** [reply]
I tested with Sky Air and it work, but I never tested VRV, but if they make x50a port, it may have a working funtion.
